### ISSUE TITLE
Problem with path

### ISSUE THREAD

**Issue by gorghi92** [Studio-42/elFinder#1342]
Hi, I ve a problem with path on connector.php page
if i use a long path i have this error: 
Invalid backend configuration.
Readable volumes not available.

My path is ../../../../../../public/ and i need to use this path

How can i do?  
ty
Andrea

**Comment by nao-pon** [comment]
@gorghi92 Please try absolute path.

**Comment by Famous-g** [comment]
Hi,
Got exactly the same problem. I tried with absolute path and it doesn't work.

My Path is ../../../../something_public/ (with 4 up '../'). I found that it works with 3 levels up but no more. If I add 4, 5 or 6, etc. levels up, it stay at 3 level up.

More precisely :
3 levels up -> OK
4 levels up -> show 3 levels up
5 levels up -> show 2 levels up !
6 levels up -> show 3 levels up
7 levels up -> show 2 levels up !
...and so on

this occurs in V2.1.11, not in V2.0 rc1

I also neeeeeed to use this path (with great V2.1.11) :/

Thank in advance.
Philippe
